### PR TITLE
Order the regions so that Other is at the bottom

### PR DESCRIPTION
**Pull request by diversemix**
Special gift for @PatFawbertMills

### CODE REVIEW

File: src/utils/sorting.ts [hunk 1]
```diff
@@ -11,3 +13,16 @@ export const sortData = (data, prop) => {
 
   return myData;
 };
+
+export const sortGeos = (data, prop) => {
+  var myData = data.sort((a, b) => {
+    if (b[prop] == "other") return -1;
```
Comment (PatFawbertMills) [inline src/utils/sorting.ts:19]: Use `===` please, double equals is generally not recommended anymore

**Review by PatFawbertMills — CHANGES_REQUESTED** [review]
Small change to use `===` over `==` please

File: src/utils/sorting.ts (hunk 1 above)
Comment (diversemix) [reply src/utils/sorting.ts:19]: Damn python indoctrination!

**Review by PatFawbertMills — APPROVED** [review]
Thanks, looks good

**Pull request merged**